ios/FluentUI/Core/Theme/FluentTheme.swift

```diff
@@ -7,17 +7,10 @@ import SwiftUI
 
 /// Base class that allows for customization of global, alias, and control tokens.
 @objc public class FluentTheme: NSObject, ObservableObject {
-    /// Initializes and returns a new `FluentTheme`, with optional custom global and alias tokens.
+    /// Initializes and returns a new `FluentTheme`.
```

Comment:
Nit: still worth it to call out that alias tokens can be customized?

Reply:
Good idea. I'll add it to the discussion field.